lintr.Rproj

```diff
@@ -19,3 +19,6 @@ BuildType: Package
 PackageUseDevtools: Yes
 PackageInstallArgs: --no-multiarch --with-keep.source
 PackageRoxygenize: rd,collate,namespace
+
+QuitChildProcessesOnExit: Yes
+DisableExecuteRprofile: Yes
```

Comment:
- Terminate [child processes](https://en.wikipedia.org/wiki/Child_process) related to active project after exiting.
- Don't run settings in `.Rprofile` to ensure reproducibility.

Reply:
Oh I see. At work, we use .Rprofile (actually Rprofile.site) to configure basic necessities so it might break things for some people.

Reply:
if so isn't that a good thing?

this only applies to R sessions launched alongside the "open RProject" action in RStudio right?

Reply:
Well I'm not so sure about that. 
e.g. R CMD check requires properly configured pandoc and, if source upstream dependencies need installing, RTools.
User configuration scripts do such things.
But I'm indifferent since I don't use RStudio.

Reply:
hmm good point... ok let's exclude this config then. Our robust CI makes up the difference to ensure reproducibility

Reply:
> At work, we use .Rprofile (actually Rprofile.site) to configure basic necessities so it might break things for some people.

But this applies only if you open the lintr project and work from inside it. I am not sure why anyone would do that.

This can cause problems. E.g. if your .Rprofile sets `options(width = 50)`, all your snapshots test outputs will be different from mine because I don't do anything similar. Not executing .Rprofile inside R package projects makes sure that individual developer's local settings don't affect package development. 

For what it's worth, I've been using these settings in every collaborative R project and have never heard of any unexpected behaviour because of this:

```r
QuitChildProcessesOnExit: Yes
DisableExecuteRprofile: Yes
```

Reply:
> This can cause problems. E.g. if your .Rprofile sets `options(width = 50)`, all your snapshots test outputs will be different from mine because I don't do anything similar.

But then our CI will not let you merge your new snapshot tests (if we ever use them)

> Not executing .Rprofile inside R package projects makes sure that individual developer's local settings don't affect package development. 

As I noted before, .Rprofile is also used to configure whatever needs configuring, such as pandoc, RTools, Proxies, CRAN mirrors.

Reply:
I have reverted the changes; they aren't that important for our development workflow.